OpenRA.Mods.Common/Activities/Parachute.cs

```diff
@@ -16,15 +16,12 @@ namespace OpenRA.Mods.Common.Activities
 {
 	public class Parachute : Activity
 	{
-		readonly IPositionable pos;
```

Comment:
Wouldn't it make sense to keep this so that we don't need to run the `is` cast every tick? (Same for the other occurrences.)

Reply:
in some locations this is still the case.
in other locations it seemed like the code would not be called too often. i.e. parachute drop.